lib/node_modules/@stdlib/stats/base/dsnanmean/benchmark/c/benchmark.length.c

```diff
@@ -96,7 +96,7 @@ static float rand_float( void ) {
 */
 static double benchmark( int iterations, int len ) {
 	double elapsed;
-	float x[ len ];
+	float x[ len ] = {0};  /* Initialize array to silence warning */
```

Comment:
This isn't quite what we want. See other similar packages which suppress the cppcheck warning instead.